docs/DeveloperGuide.md

```diff
@@ -360,6 +360,72 @@ This process is summarised in the activity diagram below
 
 ![Delete Activity Diagram](images/DeleteStudentActivityDiagram.png)
 
+#### Create Session Feature
+This section explains the implementation of the Create Session feature via the `createsession` command.
+The `CreateSessionCommand` causes the specified `Session` to be added to the Session List in the application.
+There are two compulsory fields which are the session number of the session, as well as the names of the students involved.
+
+Below is the activity diagram outlining the execution of `CreateSessionCommand`.
+
+![CreateSessionCommand activity diagram](images/CreateSessionActivityDiagram.png)
+
+Step 1:
+The Avenger(user) enters the command `createsession` and the command is parsed by the `CreateSessionCommandParser`.
```

Comment:
Perhaps try to standardise across the DG. Were the past instances using "Avenger(user)" as well, or just solely "user" or "Avenger"?